native/src/components/RemoteContent.tsx

```diff
@@ -76,13 +77,19 @@ const RemoteContent = (props: RemoteContentProps): ReactElement | null => {
 
   const onShouldStartLoadWithRequest = useCallback(
     (event: WebViewNavigation): boolean => {
+      // load allowed iframe sources on request
```

Comment:
Optional: I think the comment is not really needed as the variable names are pretty imo.
```suggestion
```